authz/rbac_translator.go

```diff
@@ -266,6 +281,63 @@ func parseRules(rules []rule, prefixName string) (map[string]*v3rbacpb.Policy, e
 	return policies, nil
 }
 
+func parseAuditLoggingOptions(options auditLoggingOptions) (*v3rbacpb.RBAC_AuditLoggingOptions, *v3rbacpb.RBAC_AuditLoggingOptions, error) {
```

Comment:
Could we have a more descriptive name? Some suggestions:
- auditLoggingOptionsToProtos()
- optionsToProtos()

Also, please document what the return values are using:
- function level comment
- by naming the return values (don't use naked returns though)

Reply:
Another option is to make this a method on the `auditLoggingOptions` struct and call it `toProtos()`

Reply:
I like it being a method on the struct and calling `toProtos()`, that feels like it gets the intention across the most